components/org.wso2.carbon.identity.conditional.auth.functions.elk/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/elk/HttpClientManager.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update the year.

Reply:
Fixed [eaf04e4](https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/98/commits/eaf04e4440a2e953eb1d5cf427ad4cda0dd0e2f2)